<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCstmGLVisualization</class>
 <widget class="QWidget" name="QCstmGLVisualization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QSplitter" name="mainSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
         <item>
          <widget class="HeatmapDisplay" name="glPlot" native="true"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,0,1,2,0">
           <item>
            <widget class="QComboBox" name="gradientSelector"/>
           </item>
           <item>
            <widget class="QCheckBox" name="outOfBoundsCheckbox">
             <property name="text">
              <string>Drop out-of-bounds</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="mouseOverLabel">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>save image</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0,0,0,0">
       <item>
        <widget class="QComboBox" name="layerSelector"/>
       </item>
       <item>
        <widget class="QCstmPlotHistogram" name="histPlot" native="true"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0,0,0">
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="frameCntr">
           <property name="text">
            <string>--/--</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="fpsLabel">
           <property name="text">
            <string>--</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="etaCntr">
           <property name="text">
            <string>---:--:--</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QCheckBox" name="summingCheckbox">
           <property name="text">
            <string>integrate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="deadpixelsinterpolationCheckbox">
           <property name="text">
            <string>interpolate low</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="highinterpolationCheckbox">
           <property name="text">
            <string>interpolate high</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="obcorrCheckbox">
           <property name="text">
            <string>OB corr</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="dropFramesCheckBox">
           <property name="toolTip">
            <string>drop a frame if there are lost packets</string>
           </property>
           <property name="text">
            <string>drop</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0" columnstretch="2,2,1,2">
         <item row="1" column="0">
          <widget class="QRadioButton" name="percentileRangeRadio">
           <property name="text">
            <string>Percentile</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>pixels fired:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="manualRangeRadio">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>manual</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QFormLayout" name="formLayout_3">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Lower Bound</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="lowerManualSpin">
             <property name="keyboardTracking">
              <bool>false</bool>
             </property>
             <property name="decimals">
              <number>0</number>
             </property>
             <property name="minimum">
              <double>-4000000000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>4000000000.000000000000000</double>
             </property>
             <property name="value">
              <double>-16384.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>upper Bound</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="upperManualSpin">
             <property name="keyboardTracking">
              <bool>false</bool>
             </property>
             <property name="decimals">
              <number>0</number>
             </property>
             <property name="minimum">
              <double>-4000000000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>4000000000.000000000000000</double>
             </property>
             <property name="value">
              <double>16384.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QFormLayout" name="formLayout_2">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Lower Bound</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="lowerPercentileSpin">
             <property name="keyboardTracking">
              <bool>false</bool>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
             <property name="value">
              <double>0.025000000000000</double>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>upper Bound</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="upperPercentileSpin">
             <property name="keyboardTracking">
              <bool>false</bool>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
             <property name="value">
              <double>0.975000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="fullRangeRadio">
           <property name="text">
            <string>Full</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>total counts:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="chargeLabel">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="countsLabel">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0,0">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="generateDataButton">
           <property name="text">
            <string>Generate Data</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="binWidthLabel">
           <property name="text">
            <string>bin count:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="binCountSpinner">
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>log</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCstmPlotHistogram</class>
   <extends>QWidget</extends>
   <header>qcstmplothistogram.h</header>
   <container>1</container>
   <slots>
    <signal>rangeChanged(QCPRange)</signal>
    <slot>changeBinSize(int)</slot>
    <slot>minClampChanged(double)</slot>
    <slot>maxClampChanged(double)</slot>
    <slot>changeRange(QCPRange)</slot>
    <slot>setActive(int)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>HeatmapDisplay</class>
   <extends>QWidget</extends>
   <header>heatmapdisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
